controllers/telegram_handlers.py

```diff
@@ -117,6 +141,15 @@ async def button_query_handler(update: Update, context: ContextTypes.DEFAULT_TYP
             WIFI_MESSAGE, reply_markup=InlineKeyboardMarkup(BUTTON_MARKUP)
         )
 
+    if "calendar" in update.callback_query.data:
+        await update.callback_query.message.reply_text(
+        'click on any of the date in calendar', reply_markup=create_calendar_markup(),
```

Comment:
```suggestion
        'Get schedule for a specific date for {month}', reply_markup=create_calendar_markup(),
```
just make sure it prints the month properly when implementing